src/main/java/homework_2/bufferReaders/IOHelper.java

```diff
@@ -0,0 +1,53 @@
+package homework_2.bufferReaders;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class IOHelper {
```

Comment:
This class is quite confusing and doesn't correlate with abstractions logic. I suggest move these methods back to initial classes